src/main/java/frc/robot/subsystems/swerve/gyroIO/GyroInterface.java

```diff
@@ -1,13 +1,16 @@
 package frc.robot.subsystems.swerve.gyroIO;
```

Comment:
I like the names you chose for interface vs sim vs physical a lot. Very clear immediately. Make sure  you're moving docstrings to the interface.